keps/NNNN-kep-template/README.md

```diff
@@ -395,50 +395,48 @@ _This section must be completed when targeting alpha to a release._
   Any change of default behavior may be surprising to users or break existing
   automations, so be extremely careful here.
 
-* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
   Also set `disable-supported` to `true` or `false` in `kep.yaml`.
-  Describe the consequences on existing workloads (e.g. if this is runtime
+  Describe the consequences on existing workloads (e.g., if this is a runtime
   feature, can it break the existing applications?).
 
 * **What happens if we reenable the feature if it was previously rolled back?**
 
 * **Are there any tests for feature enablement/disablement?**
-  The e2e framework does not currently support enabling and disabling feature
-  gates. However, unit tests in each component dealing with managing data created
-  with and without the feature are necessary. At the very least, think about
+  The e2e framework does not currently support enabling or disabling feature
+  gates. However, unit tests in each component dealing with managing data, created
+  with and without the feature, are necessary. At the very least, think about
   conversion tests if API types are being modified.
 
 ### Rollout, Upgrade and Rollback Planning
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g. what if some components will restart
-  in the middle of rollout?
+  Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
 
 * **What specific metrics should inform a rollback?**
 
-* **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
   Describe manual testing that was done and the outcomes.
   Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and do that now.
+  are missing a bunch of machinery and tooling and can't do that now.
 
-* **Is the rollout accompanied by any deprecations and/or removals of features,
-  APIs, fields of API types, flags, etc.?**
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?**
```

Comment:
Just referencing this with the previous nit on line length, they were formatted this way to keep them around 80 characters.

Reply:
I changed back the ones I remember.